packages/server/lib/controllers/sync.controller.ts

```diff
@@ -340,6 +339,20 @@ class SyncController {
             }
 
             next(err);
+        } finally {
+            const reqHeaders = getHeaders(req.headers);
```

Comment:
On your other PR you also need this + the `redacted`
Maybe you should consolidate everything with this new function

Reply:
yes that's my goal